js/mapfilter/map_pane/map_pane.js

```diff
@@ -65,68 +67,79 @@ module.exports = require('backbone').View.extend({
     var baseMaps = {}
     baseMaps[t('ui.map_pane.layers.bing')] = this.bingLayer
 
-    var tileLayers = new TileLayers()
-
-    tileLayers.fetch({
-      success: function (model, resp, opts) {
-        console.log('Successfully fetched tile layers:')
-        console.log(tileLayers)
-        tileLayers.each(function (tileLayer) {
-          console.log('+ adding a tile layer to the map:')
-          console.log(tileLayer)
-          var baseLayer = L.tileLayer('/monitoring-files/Maps/Tiles/' + tileLayer.attributes.name + '/{z}/{x}/{y}.png')
-          baseMaps[tileLayer.attributes.name] = baseLayer
-        })
-        L.control.layers(baseMaps).addTo(map)
-      },
-      error: function (model, resp, opts) {
-        console.log('Could not fetch more tile layers. Limited to Bing')
-        L.control.layers(baseMaps).addTo(map)
-      }
-    })
+    if (this.config.hasTiles()) {
```

Comment:
Syntactic sugar for `if (this.config.get('tiles'))` via that method that was created in `config.js`